packages/grafana-i18n/src/dates.ts

```diff
@@ -0,0 +1,46 @@
+// import '@formatjs/intl-durationformat/polyfill';
+import deepEqual from 'fast-deep-equal';
+import memoize from 'micro-memoize';
+
+// TODO: get types for Intl.DurationFormatOptions and Intl.DurationInput
+// from either polyfill, @types package, or by tweaking ts settings
```

Comment:
I removed the `/// @ts-expect-error` here to remove the option to accidentally commit this, forcing us to fix the types in this PR.